package.json

```
@@ -4,11 +4,12 @@
   "description": "Ubiquibot plugin template repository with TypeScript support.",
   "author": "Ubiquity DAO",
   "license": "MIT",
+  "main": "src/worker.ts",
   "engines": {
     "node": ">=20.10.0"
   },
   "scripts": {
-    "format": "run-s format:lint format:prettier format:cspell",
+    "format": "concurrently yarn:format:*",
```

```suggestion
    "format": "run-p yarn:format:*",
```

We already have run-p for this.

---

@0x4007 Sure, it's just that `run-p` comes from `npm-run-all` which received no maintenance for nearly 6 years unlike `concurrently` which also supports `yarn` and doesn't rely on `npm`. Don't mind switching it back but I think we should consider not using old packages unmaintained.
- https://www.npmjs.com/package/npm-run-all
- https://www.npmjs.com/package/concurrently

---

If it aint broke don't fix it